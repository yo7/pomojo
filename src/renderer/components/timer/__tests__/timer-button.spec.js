import {shallow} from 'vue-test-utils'
import {createRenderer} from 'vue-server-renderer'
import TimerButton from '../timer-button.vue'

describe('TimerButton', () => {
  it('renders correctly', () => {
    const renderer = createRenderer()
    const wrapper = shallow(TimerButton, {
      propsData: {running: false, onBreak: false, seconds: 1500}
    })
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) {
        throw new Error()
      }
      expect(str).toMatchSnapshot()
    })
  })
})
