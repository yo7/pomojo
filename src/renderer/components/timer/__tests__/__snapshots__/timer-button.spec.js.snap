// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimerButton renders correctly 1`] = `<div class="timer-button"><i class="icon fa fa-play"></i></div>`;
